lib/simp/packer/publish/local_dir_tree.rb

```diff
@@ -14,7 +14,20 @@ def initialize(base_dir)
           @verbose  = false
         end
 
-        def publish(box_data, copy = false)
+        # Install vagrant box into a local directory tree, generate version metadata
+        #
+        #   This places a vagrant .box file into a directory tree that is
+        #   structured more-or-less like the VagrantCloud box API.
+        #
+        #   * The .box file is placed in the versioned directory.
+        #   * metadata .json files are placed in the relevant directories.
+        #   * If the directory tree does not exist, it will be created.
+        #   * To overwite existing files/links with `:hardlink`, set the
+        #     environment variable `SIMP_PACKER_publish_force=yes`.
+        #
+        # @param [Hash] box_data The Vagrant Cloud
```

Comment:
Is there a place in which the keys for box_data are specified?

Reply:
You can see it in action at [`rakelib/vagrant.rake`](https://github.com/op-ct/simp-packer/blob/SIMP-5223-build-610-and-matrix/rakelib/vagrant.rake#L44-L47),

:ballot_box_with_check: 0314995